package.json

```diff
@@ -60,6 +60,7 @@
 		"@emotion/core": "^10.0.28",
 		"@emotion/styled": "^10.0.27",
 		"@typeform/embed": "^0.22.0",
+		"@wojtekmaj/react-datetimerange-picker": "^3.0.0",
```

Comment:
if we don't use baseui for anything else than date and timepicker maybe we should remove from deps?

Reply:
oh right, thnx 👍